src/config/app_development_dependency.go

```diff
@@ -7,6 +7,7 @@ type AppDevelopmentDependency interface {
 	GetContainerName() string
 	GetDockerConfig() (types.DockerConfig, error)
 	GetServiceEnvVariables() map[string]string
+	GetVolumeNames() []string
```

Comment:
If we're making users manually add this to the service.yml, why are dependencies exporting this.

How will this work when dependencies no longer have code?

Reply:
This is currently where are parsing the the dependency config. This would probably be moved into the compose builder when deps go away